src/graphics/aurora/modelnode.h

```diff
@@ -129,9 +130,20 @@ class ModelNode {
 		Dangly();
 	};
 
+	struct Skin {
+		float      *boneMapping;
+		uint32      boneMappingCount;
+		float      *boneWeights;
+		float      *boneMappingId;
+		ModelNode **boneNodeMap;
```

Comment:
Could you convert these arrays to `std::vector`s, please?

In general, I'd like to keep explicit new/delete to a minimum, by either using STL containers or ScopedPtr/ScopedArray.

Reply:
Fixed